simnode/generator.py

```diff
@@ -136,15 +136,16 @@ def _calculate_parallel_coords(offset: float, line_width: float) \
                         _logger.warning("Could not create line separating lanes having the same direction")
 
     def add_obstacles_to_scenario(self, scenario: Scenario) -> None:
-        from beamngpy import ProceduralCone, ProceduralCube, ProceduralCylinder, ProceduralBump
-        from dbtypes.scheme import Cone, Cube, Cylinder, Bump
-        from scipy import deg2rad
+        from beamngpy import ProceduralCone, ProceduralCube, ProceduralCylinder, ProceduralBump, StaticObject
+        from dbtypes.scheme import Cone, Cube, Cylinder, Bump, Stopsign, TrafficLightSingle, TrafficLightDouble
+        from random import randrange
         for obstacle in self.obstacles:
             obstacle_type = type(obstacle)
             height = obstacle.height
             pos = (obstacle.x, obstacle.y, height / 2.0)
-            rot = (deg2rad(obstacle.x_rot), deg2rad(obstacle.y_rot), deg2rad(obstacle.z_rot - 90))
+            rot = (obstacle.x_rot, obstacle.y_rot, obstacle.z_rot)
```

Comment:
BeamNG examples use degree instead of radiants

Reply:
Just to be safe: You are really sure it is radians? The fact that the orientation of participants used degrees and the orientation of obstacles used radians was one of the points which I wanted to mention when talking to Marc.
Maybe he already changed it in a more recent BeamNGpy version.

Reply:
https://github.com/BeamNG/BeamNGpy/blob/master/examples/object_placement.ipynb
Here, Marc is also using degrees for StaticObjects. Also, I tried rotating the objects using the degree format and the result was as expected.